eth_interface.md

```diff
@@ -67,7 +67,7 @@ Gets the hash of one of the 256 most recent complete blocks.
 
 **Returns**
 
-*nothing*
+`result` **i32** Returns 1 or 0 depending on the operation succeeded or not
```

Comment:
Does 0 mean success?

Reply:
Agree this should be clarified (clarified the ones for calls in #77).

We can either go POSIX style and have 0 as success (matches #77) or with a boolean where 1 is success (matches evm).

Reply:
I think we regret having only single failure flag in EVM 1.0. I would go with POSIX as in #77. I'm sure we will have proper discussion about this when a draft will be published.

Reply:
I wonder why EVM deviated from POSIX in the first place.